index.html

```diff
@@ -14,6 +14,56 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header class="header">
+      
+        <nav class="nav_link">
+            <img src="./img/karma-logo.svg" alt="">
+            <ul>
+                <li><a>Meet Karma</a></li>
+                <li><a>How it Works</a></li>
+                <li><a>Store</a></li>
+                <li><a>Blog</a></li>
+                <li><a>Help</a></li>
+                <li><a>Login</a></li>
+
+            </ul>
+        </nav>
+    </header>
+    <article class="main-container">
+        <h1>Introducing Karma</h1>
+        <p>Bring WIFI with you, everywhere you go.</p>
+        <button>Learn More</button>
+    </article>
+    <section class="container-services">
+        <p>Everyone nedds a little Karma</p>
```

Comment:
```suggestion
        <h2>Everyone nedds a little Karma</h2>
```
This seems like a heading so we could make it one